pycaret/tests/test_time_series.py

```diff
@@ -145,3 +145,26 @@ def test_tune_model(model, load_data):
 
     expected_period_index = load_data.iloc[-fh:].index
     assert np.all(y_pred.index == expected_period_index)
+
+
+@pytest.mark.parametrize("model", _all_models)
+def test_tune_model_random(model, load_data):
+
+    from pycaret.internal.PycaretExperiment import TimeSeriesExperiment
+    exp = TimeSeriesExperiment()
+    fh = 12
+
+    exp.setup(
+        data=load_data,
+        fold=3,
+        fh=fh,
+        fold_strategy="expandingwindow"
+    )
+
+    model_obj = exp.create_model(model)
+    tuned_model_obj = exp.tune_model(model_obj, search_algorithm="random")
+    y_pred = tuned_model_obj.predict()
```

Comment:
the `predict()` method can return an empty pandas series ?

Reply:
I doubt since the `fh` will expect some value. Have you encountered a situation like this?

Reply:
no, I was just wondering, cause in that case the first validation `isinstance(pd.Series)` will always pass

Reply:
Yes, I think, it is always returning a series for now in sktime (https://github.com/alan-turing-institute/sktime/blob/6e81f2c5d1848907a920e5d37184c64cb49fe743/sktime/forecasting/base/adapters/_tbats.py#L93)